app/uk/gov/hmrc/selfassessmentassist/v1/services/ifs/IfsService.scala

```
@@ -144,10 +144,9 @@ class IfsService @Inject()(connector: IfsConnector, currentDateTime: CurrentDate
         links = Seq(Link(riskParts(3), riskParts(4))), path = riskParts(5)))
   }
 
-  @annotation.nowarn
   private def customerTypeString(customerType: CustomerType) = customerType match {
     case TaxPayer => "Individual"
-    case Agent => "Agent"
+    case _ => "Agent"
```

I guess we should throw exception here as well case _ => throw new IllegalStateException(s"Invalid $customerType")